roles/frontend/tasks/nginx.yml

```diff
@@ -32,6 +32,13 @@
   notify:
    - reload nginx
 
+- name: install nginx conf.d/map.conf
+  copy:
+    src: nginx-map.conf
```

Comment:
Should be named nginx-conf.d-map.conf for consistency. I might actually name it prerender-map.conf when it's installed anyway.

Reply:
Fixed